about/html/edit.js

```diff
@@ -35,6 +38,9 @@ exports.create = function (api) {
 
   // TODO refactor this to use obs better
   function edit (id) {
+    // TODO - get this to wait till the connection is present !
+    var mutual = Mutual.init(api.sbot.obs.connection())
```

Comment:
@dangerousbeans here was your problem - you were init'ing with `api.sbot` - this is a socket (an object which in our case had keys async and pull, not what you want).

`api.sbot.obs.connection` is an observeable which resolves to sbot when it's connected (and d_dangerously_ to null when it's not). This is not a beautiful solution and will need fixing in the future, but we can generally be sure that the sbot is connected by the time a person is browsing to a profile picture.

WORKS FOR ME!   :sob:

Reply:
sweet thanks!
I had little idea of what I was doing so just found the first vaguely sbot-like thing I could find >.<